src/components/Sections.vue

```diff
@@ -298,9 +298,9 @@ color: white
 </style>
 <script>
 import WebImg from '../assets/img/web-light.svg'
-import MobileImg from '../assets/img/mobile-dark.svg'
+const MobileImg = import('../assets/img/mobile-dark.svg')
```

Comment:
Confused by this. If this change is needed then why not the line above? Please explain.